src/text-tags.js

```diff
@@ -124,7 +124,7 @@ function getSeoData(config, pageType, data, url = {}, seoConfig = {}) {
     const { sections = [] } = config;
     const section = sections.find(section => ownerType == 'section' && section.id === ownerId) || {};
     const customSeo = get(data, ["data", "customSeo"], {})
-    if (seoMetadata.data || section.id) {
+    if (seoMetadata.data || section.id || !isEmpty(customSeo)) {
```

Comment:
@Deokumarjnu what will be the `customSeo['page-title']` if customseo is not empty and `customSeo['page-title']` is not present or the same with customSeo.title

Reply:
This function only called on the home page and section page, also this check only failed for the home page because `seometadata` was empty, if nothing will be defined `seoMetadata` and `customSeo` then it will render undefined.  ideally, every publisher used to give some SEO meta tags for prod.

https://github.com/quintype/quintype-node-seo/blob/master/src/text-tags.js#L129-L133

Here is the logic, 
```
'page-title': customSeo['page-title'] || section.name,
 title: customSeo.title || section.name,

```